src/arearange/area-range-component.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from "react";
+import * as Highcharts from "highcharts";
+import more from "highcharts/highcharts-more";
+import HighchartsReact from "highcharts-react-official";
+import { getRangeData } from "../service";
+import { AreaRangeData } from "./area-range-data";
+import { toAreaRangeSeries } from "../util";
+more(Highcharts);
+
+interface AreaRangeProps {
+    type: string;
+    stateCode: string;
+}
+
+interface AreaRangeState {
+    optionsForAllCharts: Highcharts.Options[];
+}
+
+const optionsDelegate = (rangeData: AreaRangeData): Highcharts.Options => {
+    return {
+        title: {
+            text: rangeData.chartingMetadata.title,
+        },
+        xAxis: {
+            type: "datetime",
+            dateTimeLabelFormats: {
+                day: "%e %b",
+                month: "%b%y",
+            },
+            title: {
+                text: rangeData.chartingMetadata.xAxisLabel,
+            },
+        },
+
+        yAxis: {
+            type: "logarithmic",
```

Comment:
as discussed in slack, we don't see log.

Reply:
okie, i moved to show logarithm as per the email from Jeffrey. I can remove it